### PR TITLE
Added Anomaly Visualization Workbook and Anomaly Data workbook

### PR DESCRIPTION
**Pull request by jyp033**
Required items, please complete
Change(s):

Added Anomaly Visualization Workbook to visualize anomaly and incident data to the user
Added Anomaly Data workbook to show data, Incidents, and hunting bookmarks for a specific anomaly
Reason for Change(s):

Added as part of an effort to give users a better interface for investigating anomaly data
Version Updated:

New Workbook
Testing Completed:

Yes
Checked that the validations are passing and have addressed any issues that are present:

Yes

### CODE REVIEW

**Comment by jyp033** [comment]
@alex.karabasevic@microsoft.com

**Review by v-atulyadav — COMMENTED** [review]
Hi @jyp033, please resolve conflicts.
image

**Comment by jyp033** [comment]
Sorry about that, files have been merged

**Review by v-atulyadav — COMMENTED** [review]
Hi @jyp033, please add dataTypesDependencies in WorkbooksMetadata.json file

image

File: Workbooks/WorkbooksMetadata.json [hunk 1]
```diff
@@ -1599,5 +1599,31 @@
         "templateRelativePath": "MicrosoftSentinelDeploymentandMigrationTracker.json",
         "subtitle": "",
         "provider": "Microsoft Sentinel Community"
+      },
+      {
+        "workbookKey": "AnomaliesVisualization",
```
Comment (v-atulyadav) [inline Workbooks/WorkbooksMetadata.json:1604]: Please update workbookkey property value as "AnomaliesVisualizationWorkbook"

File: Workbooks/WorkbooksMetadata.json [hunk 2]
```diff
@@ -1599,5 +1599,31 @@
...
+        "dataConnectorsDependencies": [],
+        "previewImagesFileNames": [ "AnomaliesVisualizationWorkbookWhite.png", "AnomaliesVisualizationWorkbookBlack.png" ],
+        "version": "1.0.0",
+        "title": "AnomaliesVisulization",
+        "templateRelativePath": "AnomaliesVisulization.json",
+        "subtitle": "",
+        "provider": "Microsoft Sentinel Community"
+      },
+      {
+        "workbookKey": "AnomalyData",
```
Comment (v-atulyadav) [inline Workbooks/WorkbooksMetadata.json:1617]: Please update workbookkey property value as "AnomalyDataWorkbook"

**Review by v-atulyadav — CHANGES_REQUESTED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Review by v-rucdu — APPROVED** [review]

**Pull request merged**